### DISCUSSION TITLE
【灌水】昨天起白祇重工求职，倒是成功了，但我还是放弃了

### DISCUSSION THREAD

**Discussion by BiaoBiaoB** [share121/inter-knot#589] [其他]
![Zenless Zone Zero Screenshot 2024 08 09 - 14 41 21 86](https://github.com/user-attachments/assets/2a183406-6d51-4e6d-8530-cc5373e4ff32)

昨天我去白祇重工求职，原本成功的时候我还是很开心的。结果在路上的时候遇到了一个叫做什么兄弟布的邦布，那个邦布一直在喊着兄弟，我凑近之后那个家伙就一直在喊着：兄弟你好香。搞得我有点害怕，最后还是放弃了这份相当不错的会计工作

**Comment by FrostPhoenix79** [comment]
照片是你本人还是那个邦布捏？如果是本人的话建议加个联系方式吗？兄弟，你好香:D

**Comment by GUILAISHIXV** [comment]
兄弟转下身，有急事
